datafusion/core/src/datasource/physical_plan/parquet/opener.rs

```diff
@@ -41,21 +41,41 @@ use std::sync::Arc;
 
 /// Implements [`FileOpener`] for a parquet file
 pub(super) struct ParquetOpener {
+    /// Execution partition index
     pub partition_index: usize,
+    /// Column indexes in `table_schema` needed by the query
```

Comment:
This in particular was unclear when @itsjunetime  and I read the code (were the indexes in terms of file_schema or table_schema?) and was the impetus for this PR